docs/update-go-version.md

```diff
@@ -53,7 +53,4 @@ In addition, [go rules for bazel](https://github.com/bazelbuild/rules_go) have t
     ```
 * Visit [Bazel's releases page](https://github.com/bazelbuild/rules_go/releases) and check whether the current Bazel version supports the new Go version.
   * If it is not supported, replace the `io_bazel_rules_go` definition with the one provided in Bazel's page.
-* Use [project-infra's uploader tool](https://github.com/kubevirt/project-infra/blob/master/plugins/cmd/uploader/README.md) to upload new dependencies to dependency mirror.
```

Comment:
Thanks for catching this. I think this is a result of changing the default branch to main.

@fossedihelm you are currently working on this also, did you already have a PR on this which we can relate to?

Reply:
Yes, [#6814](https://github.com/kubevirt/kubevirt/pull/6814) already address this issue. Thanks.